tidy3d/components/lumped_element.py

```diff
@@ -281,19 +335,461 @@ def to_structure(self) -> Structure:
             "tt": Medium(conductivity=conductivity),
             "ss": Medium(conductivity=conductivity),
         }
+        return Structure(
+            geometry=self.to_geometry,
+            medium=Medium2D(**medium_dict),
+        )
+
+    @cached_property
+    def to_geometry(self) -> ClipOperation:
+        """Converts the :class:`CoaxialLumpedResistor` object to a :class:`Geometry`."""
         rout = self.outer_diameter / 2
         rin = self.inner_diameter / 2
         disk_out = Cylinder(axis=self.normal_axis, radius=rout, length=0, center=self.center)
         disk_in = Cylinder(axis=self.normal_axis, radius=rin, length=0, center=self.center)
         annulus = ClipOperation(operation="difference", geometry_a=disk_out, geometry_b=disk_in)
+        return annulus
+
+
+class NetworkConversions(Tidy3dBaseModel):
+    """Helper functionality for directly computing complex conductivity and permittivities using
+    equations in _`[1]`. Useful for testing the direct translations of lumped network parameters into
+    an equivalent PoleResidue medium.
+
+    Notes
+    -----
+
+        This implementation follows a similar approach as _`[1]` with a couple small differences. Instead of
+        scaling the complex conductivity by the size of a single grid cell, we later scale the quantities by the
+        size of the lumped element in the FDTD simulation. In many cases, we will assume the time step is small,
+        so that the complex conductivity can be expressed more simply as a rational expression.
+
+        **References**
+
+        .. [1]  J. A. Pereda, F. Alimenti, P. Mezzanotte, L. Roselli and R. Sorrentino, "A new algorithm
+                for the incorporation of arbitrary linear lumped networks into FDTD simulators," IEEE
+                Trans. Microw. Theory Tech., vol. 47, no. 6, pp. 943-949, Jun. 1999.
+    """
+
+    @staticmethod
+    def complex_conductivity(
+        a: tuple[float, ...], b: tuple[float, ...], freqs: np.ndarray, dt: float = None
+    ):
+        """Returns the equivalent conductivity of the lumped network over the range of frequencies
+        provided in ``freqs`` using the expression in _`[1]`.
+
+        Parameters
+        ----------
+        a : tuple[float, ...]
+            Coefficients of the numerator polynomial
+        b : tuple[float, ...]
+            Coefficients of the denominator polynomial.
+        freqs: np.ndarray
+            Frequencies at which to evaluate model.
+        dt : float
+            Time step of the FDTD simulation.
+
+        Returns
+        -------
+        np.ndarray
+            The resulting complex conductivity.
+        """
+
+        if dt is None:
+            # If the FDTD time step is not provided, use the limiting case of dt -> 0.
+            # In many cases, this approximation is quite accurate.
+            K_tan = -1j * 2 * np.pi * freqs
+        else:
+            K_tan = -1j * (2 / dt) * np.tan(2 * np.pi * freqs * dt / 2)
```

Comment:
It looks ADE implementation of CCPR model is assuming dt->0. I wonder when will the numerical conductivity here be useful.

Reply:
This is left over from my initial implementation and I kept it for easy comparison to the reference paper and possibly future testing in case it is needed. But, it is probably not needed.